### DISCUSSION TITLE
AI and LLMs

### DISCUSSION THREAD

**Discussion by trey-stafford** [nsidc/tech-radar#45] [Tools]
What are the policies and restrictions related to our use of AI?

What is the NASA policy and how does it relate to us as contractors? The NASA policy stipulates being careful with sensitive data, for example. Annual security training also included information about AI and contributions to code.

What's CU policy? CIRES/NSIDC's?

Are there some use-cases that are more allowed than others?

Could these be used used by management? E.g., for ASAs?

**Comment by eigenbeam** [comment]
Two specific tools I'm experimenting with:

- [GitHub Copilot](https://github.com/features/copilot)
- [CodiumAI](https://www.codium.ai/)
